### PR TITLE
Feature web session handling

### PR DESCRIPTION
**Pull request by linolarios**
Was added a Interceptor to check if the session stills alive otherwise the user is 
redirect to the login page

### CODE REVIEW

File: src/main/groovy/com/nearsoft/happinessometer/web/HappinessometerWebApplication.groovy [hunk 1]
```diff
@@ -17,6 +16,11 @@ class HappinessometerWebApplication extends WebMvcConfigurerAdapter {
         viewControllerRegistry.addViewController("/").setViewName("index")
     }
 
+
```
Comment (abadongutierrez) [inline src/main/groovy/com/nearsoft/happinessometer/web/HappinessometerWebApplication.groovy]: I think this extra line does not make sense. Can you remove it?

File: src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy [hunk 2]
```diff
@@ -18,11 +18,11 @@ import wslite.rest.RESTClient
 
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpSession
-import javax.validation.Valid
 
 @Controller
 class LoginController extends BaseController {
     static final Logger logger = LoggerFactory.getLogger(LoginController)
+    static final int MAX_TIME_SESSION_VALUE = 60*60;
```
Comment (abadongutierrez) [inline src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy]: lets try with an open session of 12 hours

Comment (linolarios) [inline src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy]: all right

**Comment by linolarios** [comment]
https://github.com/Nearsoft/happinessometer-web/commit/3cd60640b46b5dbae98c83c04a781efce9cf16ca

**Comment by abadongutierrez** [comment]
@linolarios please "squash and merge" this changes! 👍

**Pull request merged**